lnwallet/btcwallet/psbt.go

```diff
@@ -85,6 +110,7 @@ func (b *BtcWallet) FundPsbt(packet *psbt.Packet, minConfs int32,
 	return b.wallet.FundPsbt(
 		packet, keyScope, minConfs, accountNum, feeSatPerKB,
 		b.cfg.CoinSelectionStrategy,
+		wallet.WithCustomChangeScope(changeScope),
```

Comment:
Since passing `wallet.WithCustomChangeScope(nil)` is equal to not passing an option at all, we _could_ make the default case (change scope bein `nil`) a bit more explicit here by doing:

```
var opts []wallet.TxCreateOption
if changeScope != nil {
    opts = append(opts, wallet.WithCustomChangeScope(changeScope))
}
return b.wallet.FundPsbt(
    ...,
    opts...,
)
...

```
Not sure if that makes it _that_ more clear...

Reply:
Interesting! Indeed it is clearer but now, change scope can never be nil:
 If  it was nil in `(b *BtcWallet) FundPsbt`, then we will use `changeScope = &waddrmgr.KeyScopeBIP0084` for default and imported scopes (see [l82](https://github.com/lightningnetwork/lnd/pull/7348/files/3b53f47f9d7de7af13b4fc48f6ca9c766ecaec9d..b6a2fb034e758234ea9ac9d9059e900b562c8f82#diff-f92ee065de0a8d5d61de2f8e08f4c31b37611ed54edc4b8522984735d88e7540R82) and l89). 
For custom accounts will we use the same key scope as the selection coin (never nil)

But adding this doesn't do any harm!